src/main/java/mate/academy/model/Role.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "roles")
+public class Role {
```

Comment:
you can implement `GrantedAuthority` here in order to simplify the setup

Reply:
ok